packages/react/src/index.ts

```diff
@@ -1,25 +1,46 @@
+import { type Hume } from 'hume';
+
 export * from './lib/useSoundPlayer';
 export * from './lib/useMicrophone';
 export * from './lib/VoiceProvider';
 export * from './lib/useVoiceClient';
 export * from './lib/connection-message';
 
 export {
-  type SocketConfig,
   Channels,
-  TTSService,
   AudioEncoding,
-  type AssistantEndMessage,
-  type AssistantTranscriptMessage,
-  type AudioMessage,
-  type AudioOutputMessage,
-  type JSONMessage,
+  TTSService,
   type TimeSlice,
-  type UserInterruptionMessage,
-  type UserTranscriptMessage,
-  type ToolCall,
-  type ToolResponse,
-  type ToolError,
-  type ChatMetadataMessage,
   LanguageModelOption,
-} from '@humeai/voice';
+} from './types';
+
+export type SocketConfig = Hume.empathicVoice.chat.Chat.ConnectArgs &
+  Hume.empathicVoice.chat.Chat.Options;
+
+export type {
+  AssistantEnd as AssistantEndMessage,
+  AudioInput as AudioMessage,
+  AudioOutput as AudioOutputMessage,
+  UserInterruption as UserInterruptionMessage,
+  AssistantMessage as AssistantTranscriptMessage,
+  SubscribeEvent as JSONMessage,
+  UserMessage as UserTranscriptMessage,
+  ToolCallMessage as ToolCall,
+  ToolResponseMessage as ToolResponse,
+  ToolErrorMessage as ToolError,
+  ChatMetadata as ChatMetadataMessage,
+} from 'hume/api/resources/empathicVoice';
```

Comment:
I wonder if we should just commit to making a breaking change for these type names instead of changing them to maintain backwards compat. @gregorybchris @zachkrall @zgreathouse any opinions here?

Reply:
Normally I'd agree that we should invest in standardizing across packages earlier rather than later, but I think it may make sense to incur tech debt at the react SDK level for the sake of minimizing friction in integrations downstream. Whatever we break here we have to migrate everywhere. No strong opinions though.

Reply:
We're gonna need to do a migration guide anyway.. would defer to you and the Zachs

Reply:
yeah, I'm open-minded if people think otherwise, but my personal opinion is that if we eventually want to make the breaking change, we should just do it now.

Reply:
@yinishi it may be worth releasing a beta version with the types as back compat and then doing a fast follow to keep the ball moving, curious for your thoughts

Reply:
sure, works for me, down to keep things moving forward